samples/CppArExample/Android/Android.pri

```diff
@@ -0,0 +1,37 @@
+#-------------------------------------------------
```

Comment:
General comment - this should probably go under the `Examples` folder and not introduce a new `samples` folder. They are for the same purpose

Reply:
I will move the examples in the correct directory in the next PR, to simplify the following of the comments.

Reply:
@ldanzinger the `Examples` folder is not a folder with a list of examples but it contains one project called `Examples`. If the AR examples are moved to this folder, I suppose the `Examples` project should be rename and moved to a subdir. Which name do you want to use?

Reply:
@GuillaumeBelz I'd say leave the example project as-is, and then add a sub dir called "AR" and put your AR samples in there